### PR TITLE
Make Pip resolution repository editable

### PR DESCRIPTION
**Pull request by RobiNino**
- [x] All [tests](https://github.com/jfrog/jfrog-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: tasks/JFrogPip/task.json [hunk 1]
```diff
@@ -65,7 +65,10 @@
             "defaultValue": "",
             "required": true,
             "visibleRule": "command = install",
-            "helpMarkDown": "Sets the resolution repository."
+            "helpMarkDown": "Sets the resolution repository.",
+            "properties": {
+                "EditableOptions": "True"
+            }
```
Comment (eyalbe4) [inline tasks/JFrogPip/task.json:71]: I suggest we make all repository selections editable, in all tasks, including the deployment repositories selections.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: tasks/JFrogPip/task.json (hunk 1 above)
Comment (RobiNino) [reply tasks/JFrogPip/task.json:71]: They are all already editable, I validated it and this was the only exception

Comment (eyalbe4) [reply tasks/JFrogPip/task.json:71]: I noticed this one @RobiNino -
```
        {
            "name": "targetResolveRepo",
            "type": "pickList",
            "label": "Resolution repository",
            "defaultValue": "",
            "required": true,
            "visibleRule": "command = install",
            "helpMarkDown": "Sets the resolution repository."
        },
```
Inside JFrogPip/task.json

Comment (eyalbe4) [reply tasks/JFrogPip/task.json:71]: OK. My bad probably.

**Pull request merged**